intro_to_python/intro_to_python.md

```diff
@@ -259,7 +297,7 @@ You can download Python directly to your computer, use Anaconda to download Pyth
 </div>
 ***********************************************************************
 
-### Additional Resources
+## Additional Resources
 
 For a beginner-friendly walk-through of writing Python scripts, [Hello World! Your Very First Computer Program](https://youtu.be/cV-QwT4fV7M) delivered by Jeff Pennington is an excellent introduction if you've got an hour to spend.
 
```

Comment:
Joy's article about cloud tools might be too CHOP specific for our ultimately larger audience. Maybe you want to link directly to her notebook instead?

Reply:
I see what you mean, there is some CHOP-specific stuff at the end. I worry thought the notebook on its own might not have enough information to be helpful... I'll look around for better resource.

Reply:
Done! Added the notebook but also a section about Google Colab, which makes it make sense (I hope).